mitiq/pec/representations/tests/test_depolarizing.py

```diff
@@ -167,3 +172,80 @@ def test_three_qubit_local_depolarizing_representation_error():
         represent_operation_with_local_depolarizing_noise(
             Circuit(CCNOT(q0, q1, q2)), 0.05,
         )
+
+
+@pytest.mark.parametrize("circuit_type", ["cirq", "qiskit", "pyquil"])
```

Comment:
Is this used?

Reply:
Good catch! I fixed this. Now circuit_type is used.